README.md

```diff
@@ -22,7 +22,7 @@ The table below shows developers details - names, github username linking to the
 | Number | Name | Github Username | Favourite Programming Language |
 | - | ----------- | ------------------------------ | ----- |
 | 1 | Segun Ajibola | [segunajibola](https://github.com/segunajibola) | JavaScript |
-| 2 | Add your name | [Add your github username](https://github.com/<github-username>) | Add your favourite programming language |

```

Comment:
Can you make the previous number 2 be the new number 3?

Reply:
Sure, I would do that and revert back by creating a PR